src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,15 +34,16 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart = shoppingCartService.getByUser(userService
+                .findByEmail(authentication.getName()).get());
         return orderResponseDtoMapper.mapToDto(orderService.completeOrder(cart));
     }
 
     @GetMapping
-    public List<OrderResponseDto> getOrderHistory(@RequestParam Long userId) {
-        return orderService.getOrdersHistory(userService.get(userId))
-                .stream()
+    public List<OrderResponseDto> getOrderHistory(Authentication authentication) {
+        return orderService.getOrdersHistory(userService
+                        .findByEmail(authentication.getName()).get()).stream()
                 .map(orderResponseDtoMapper::mapToDto)
```

Comment:
This code may look much pretty:
```suggestion
        User user = userService.findByEmail(authentication.getName())
        	.orElseThrow(() -> EntityNotFoundException("Can't get user by email: " + authentication.getName()));
        return orderService.getOrdersHistory(user).stream()
                .map(orderResponseDtoMapper::mapToDto)
```